store/src/main/java/com/onlineStore/coherent/multithreading/ThreadAddProductsInOrder.java

```diff
@@ -1,30 +1,29 @@
 package com.onlineStore.coherent.multithreading;
 
-import database.DataBase;
-import product.Product;
+import com.onlineShop.coherent.order.Order;
+import com.onlineShop.coherent.product.Product;
 
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 
 public class ThreadAddProductsInOrder extends Thread {
     List<Product> productsToOrder;
```

Comment:
This variable is not needed anymore and should be removed. If you need the list of products in the order , you should get them from database

Reply:
Remove this variable